### ISSUE TITLE
Error: createLeafletElement() must be implemented - Leaflet Upgrade

### ISSUE THREAD

**Issue by DenizTemizer** [OpenGov/react-leaflet-cluster-layer#7]
Since the update of Leaflet to v 1.1 a component must contain a createLeafletElement function otherwise MapLayer.js will throw an error.

```javascript
// eslint-disable-next-line no-unused-vars
MapLayer.prototype.createLeafletElement = function createLeafletElement(props) {
    throw new Error('createLeafletElement() must be implemented');
};
```

I fixed this error with implementing a createLeafletElement function in the ClusterLayer.js.

```javascript
ClusterLayer.prototype.createLeafletElement = function createLeafletElement() {};
```

[How to upgrade Leaflet.](https://github.com/PaulLeCam/react-leaflet/blob/master/UPGRADING.md#v11)

**Comment by texas697** [comment]
I am trying to implement your solution but I am now getting this error.
TypeError: Cannot read property 'getPanes' of undefined
can you help @DenizTemizer ?

**Comment by DoubleYo** [comment]
Fix by PR #9

**Comment by vasiledraghita** [comment]
@DoubleYo I am using the library with your updated version 0.0.4 and I receive this error ClusterLayer.js:113 Uncaught TypeError: Cannot read property 'getPanes' of undefined. Should I send other props to ClusterLayer other than markers and clusterComponent? I am using this with these dependencies: 
{
"leaflet": "^1.3.4",
"leaflet.markercluster": "^1.3.0",
"react": "^16.2.0",
"react-dom": "^16.2.0",
"react-leaflet": "^2.0.1",
"react-leaflet-cluster-layer": "git+https://github.com/DoubleYo/react-leaflet-cluster-layer.git#master",
}
